firefox_ios/views/client_counts.view.lkml

```diff
@@ -0,0 +1,60 @@
+include: "//looker-hub/firefox_ios/views/baseline_clients_daily_table.view.lkml"
+
+view: client_counts {
+  extends: [baseline_clients_daily_table]
+
+  dimension_group: since_first_seen {
+    type: duration
+    description: "Amount of time that has passed since the client was first seen."
+    sql_start: ${TABLE}.first_seen_date ;;
+    sql_end: ${TABLE}.submission_date ;;
+    intervals: [
+      day, week, month, year
+    ]
+  }
+
+  dimension: have_completed_period {
+    type: yesno
+    description: "
+      Only for use with cohort analysis.
+      Filter on true to remove the tail of incomplete data from cohorts.
+      Indicates whether the cohort for this row have all had a chance to complete this interval.
+      For example, new clients from yesterday have not all had a chance to send a ping for today."
+    sql:
+      DATE_ADD(
+        {% if client_counts.first_seen_date._is_selected %}
+          DATE_ADD(DATE(${client_counts.first_seen_date}), INTERVAL 1 DAY)
+        {% elsif client_counts.first_seen_week._is_selected %}
+          DATE_ADD(DATE(${client_counts.first_seen_week}), INTERVAL 1 WEEK)
+        {% elsif client_counts.first_seen_month._is_selected %}
+          DATE_ADD(DATE(${client_counts.first_seen_month}), INTERVAL 1 MONTH)
```

Comment:
I tried to pivot on first_seen_month and got an error about `2021-03` being an invalid timestamp format.

Reply:
Hmm, I didn't realize Looker would truncate the date string like that. Let's see how we can handle it.

Reply:
Okay, these are fixed now.